data/data/aws/main.tf

```diff
@@ -52,8 +52,8 @@ module "iam" {
   tags = "${local.tags}"
 }
 
-module "dns" {
-  source = "./route53"
+module "dns-api" {
+  source = "./route53/api"
```

Comment:
I think this is an opportunity to resolve the `dns` vs. `route53` naming difference.  I don't care which we standardize on, but I'd like to have the module name match the source path (e.g. `route53-api` and `./route53/api` or `dns-api` and `./dns/api`).

Reply:
IMO the current naming seems fine to me. @wking can you expand on why you think `but I'd like to have the module name match the source path` is something we should strive for.

Reply:
I just don't see a point to having two names for the same thing.